integration-test/action.yaml

```diff
@@ -0,0 +1,35 @@
+name: "golang: Integration-Test"
+description: "Conditionally executes golang integration tests via docker, if docker compose file is found"
+inputs:
+  checkout-repo:
+    required: false
+    description: "Perform checkout as first step of action"
+    default: "true"
+  github-token:
+    required: true
+    description: "GitHub token for docker compose. e.g. 'secrets.GITHUB_TOKEN'."
+  docker-compose-file:
+    required: false
+    description: "Relative or absolute path to docker-compose.yaml file"
+    default: ./docker-compose.yaml
+runs:
+  using: composite
+  steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      if: inputs.checkout-repo
+      with:
+        fetch-depth: 0
+    - name: Build and Test
```

Comment:
We're naming the things with Sentence case.

Reply:
Ah. I made a pass through all of the action.yaml files to correct this